geomstats/visualization.py

```diff
@@ -490,6 +494,221 @@ def draw(self, ax, **kwargs):
         ax.scatter(translation[:, 0], translation[:, 1], s=16, **kwargs)
 
 
+class KendallSphere:
+    """Class used to plot points in Kendall shape space of 2D triangles.
+
+    David G. Kendall showed that the shape space of 2D triangles is isometric
+    to the 2-sphere of radius 1/2 [Kend]. This class encodes this isometric
+    representation, offering a 3D visualization of Kendall shape space, and its
+    related objects.
+
+    Attributes
+    ----------
+    points : list
+        List of points to plot on the Kendall sphere.
+    point_type : str
+        Type of the points. Can be either 'pre-shape' (for points in Kendall
+        pre-shape space) or 'extrinsic' (for points given as 3x2 matrices).
+        Optional, default: 'pre-shape'.
+    pole : array-like, shape=[3,2]
+        Equilateral triangle (north pole).
+    ua : array-like, shape=[3,2]
+        Tangent vector toward isocele triangle at vertex A.
+    ub : array-like, shape=[3,2]
+        Tangent vector toward isocele triangle at vertex B.
+    na : array-like, shape=[3,2]
+        Tangent vector such that (ua,na) is a positively oriented
+        orthonormal basis of the horizontal space at north pole.
+
+    References
+    ----------
+    [Kend]  David G. Kendall.
```

Comment:
I think the syntax should be something like ..[Kend] (see other references in other .py files): this allows the rendering to be pretty on the documentation website